packages/extension-points/README.md

```diff
@@ -35,3 +35,14 @@ widgets:
     component: js/build/options_buttons.widget.js
     multiple: true
 ```
+
+### Routes Configuration
+
+A single `*.admin_ui.routes.yml` file can contain as many routes as needed. Note that route will be prefixed with the Drupal module name. For example the following route would be accessible at, `/admin_ui_widget_example/test`.
```

Comment:
Makes more sense now. Do you think it would make sense to add a mention that the following code is located in `admin_ui_widget_example` module?

Reply:
👍